my-new-webapp-servlet/src/main/java/servlet/listener/PropertiesContextListener.java

```diff
@@ -0,0 +1,47 @@
+package servlet.listener;
+
+import org.apache.log4j.Logger;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+
+public class PropertiesContextListener implements ServletContextListener {
+
+    static final Logger LOGGER = Logger.getLogger(PropertiesContextListener.class);
+
+    @Override
+    public void contextInitialized(ServletContextEvent sce) {
+
+        final ServletContext servletContext = sce.getServletContext();
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+
+        String productionProperties = "/usr/local/my-web-app/production.properties";
+
+        try {
+
+            LOGGER.info("Trying use production properties by path: "+productionProperties);
+            File file = new File(productionProperties);
+            if(!file.exists()){
+                LOGGER.info("Production properties by path: "+productionProperties+" is not existing.");
+                file = new File("config.properties");
+                LOGGER.info("Trying inner properties of MyNewWebApp by path: "+classLoader.getResourceAsStream(file.getPath()));
```

Comment:
Let's extract it into variable and put after `productionProperties` on row 24-25